product_docs/docs/pgd/5/quickstart/quick_start_docker.mdx

```diff
@@ -197,7 +197,7 @@ less democluster/config.yml
 
 ### Deploying the cluster
 
-You can now [deploy](/tpa/latest/tpaexec-deploy/) the distributed cluster. The `tpaexec` command will automatically run its `provision` command to prepare the Docker containers for the deployment:
+You can now [deploy](/tpa/latest/tpaexec-deploy/) the distributed cluster. The `tpaexec` command runs its `provision` command to prepare the Docker containers for the deployment:
```

Comment:
```suggestion
You can now [deploy](/tpa/latest/tpaexec-deploy/) the distributed cluster. 
For Docker deployments, this will both provision the required Docker containers and deploy the software to those containers:
```